infrahouse_toolkit/aws/tests/config/test_get_account_id.py

```diff
@@ -6,7 +6,7 @@
 
 CONTENT = dedent(
     """
-    [default]
+    [profile default]
```

Comment:
I don't believe this is a valid aws config. 
https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-files.html#cli-configure-files-format